streams.json

```diff
@@ -8780,24 +11613,28 @@
               "dlloyd@redhat.com"
           ],
           "repository_type": "GIT",
-          "repository_url": "https://github.com/jbossas/staxmapper",
+          "repository_url": "https://github.com/jbossas/staxmapper.git",
           "codebase": "master",
-          "tag": "1.2.0.Final",
+          "tag": "jboss-parent-16",
+          "group_id": "org.jboss",
+          "artifact_id": "staxmapper",
           "version": "1.2.0.Final-redhat-1",
-          "gav": "org.jboss",
+          "gav": "staxmapper:org.jboss",
           "comment": " master is at 1.4.0, there is no maintenance branch"
         },
         {
           "component_name": "jbossas-wildfly-core-eap",
           "contacts": [
               "bstansberry@redhat.com"
           ],
-          "repository_type": "GIT",
-          "repository_url": "https://github.com/jbossas/wildfly-core-eap",
+          "repository_type": "",
+          "repository_url": "",
```

Comment:
looks wrong, this only removes repository_url of component jbossas-wildfly-core-eap

Reply:
@soul2zimate Fixed, we had this one as wildfly-core, I've added an entry in the dictionary so we can see it as jbossas-wildfly-core-eap as well